jetty-io/src/main/java/org/eclipse/jetty/io/ssl/SslConnection.java

```diff
@@ -979,10 +993,10 @@ public boolean flush(ByteBuffer... appOuts) throws IOException
             }
             catch (Throwable x)
             {
-                if (LOG.isDebugEnabled())
-                    LOG.debug(SslConnection.this.toString(), x);
                 close(x);
-                throw x;
+                rethrow(x);
```

Comment:
You could make this a little nicer with just:
```java
throw rethrow(x);
```
Ie the method returns the IOException, but itself will throw Runtime or Error if need be..... hmmmm maybe that's not better.